### PR TITLE
kraken: when fetching tradable pairs filter by status

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Presence of API restricted currency pair which is now filtered out. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [x] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/kraken/kraken_wrapper.go [hunk 1]
```diff
@@ -357,24 +357,27 @@ func (k *Kraken) FetchTradablePairs(ctx context.Context, a asset.Item) (currency
...
-		for i := range symbols {
-			if strings.Contains(symbols[i].Altname, ".d") {
+		for _, info := range symbols {
+			if info.Status != "online" {
 				continue
 			}
-			base := assetTranslator.LookupAltname(symbols[i].Base)
+			if strings.Contains(info.Altname, ".d") {
+				continue
+			}
```
Comment (gloriousCode) [inline exchanges/kraken/kraken_wrapper.go:366]: Could you point me to why you've added this? It doesn't get hit, I don't see it in their documentation, nor was it in my old un-updated config

Comment (shazbert) [reply exchanges/kraken/kraken_wrapper.go:366]: I didn't it was already there. No idea, will remove.

Comment (thrasher-) [reply exchanges/kraken/kraken_wrapper.go:366]: Previously Kraken use to add ".d" to the altname if a trading pair was disabled, before the status field was implemented

**Review by gloriousCode — APPROVED** [review]
Lovely. tACK

**Review by thrasher- — APPROVED** [review]

**Pull request merged**